[Proposal]  Remove implicit truncations. (#165)
chipsalliance/firrtl-spec: spec.md

The `AsyncReset`{.firrtl} type can be connected to another `AsyncReset`{.firrtl} or to a `Reset`{.firrtl}.

Two enumeration types may be connected if both have the same number of variants, and both the enumerations' i'th variants have matching names and equivalent types.

Two vector types may be connected if they have the same length, and if their element types are connectable.

> Two bundle types may be connected if they have the same number of fields, and both the bundles' i'th fields have matching names and orientations, as well as connectable types.

Reviewer: ```suggestion
Two bundle types may be connected if they have the same number of fields, and both the bundles' i'th fields have matching names and orientations, and are connectable types.
```
